testsuite/features/srv_change_task_schedule.feature

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2015-2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Change the schedule of task mgr-sync-refresh-default
+
+  Scenario: Change the schedule of task mgr-sync-refresh-default
+    Given I am authorized as "admin" with password "admin"
+    When I follow "Admin"
+    And I follow "Task Schedules"
+    And I follow "mgr-sync-refresh-default"
+    And I check radio button "weekly"
+    And I select "Friday" from "date_day_week"
+    And I click on "Update Schedule"
+    Then I should see a "Schedule mgr-sync-refresh-default has been updated." text
+    When I follow "Task Schedules"
+    And I follow "mgr-sync-refresh-default"
+    Then I should see a "Friday" text
+    And radio button "weekly" is checked
```

Comment:
@lkotek : this step should be named "And radio button ... should be checked"

@mantel : it's outside of the scope of your PR, please ignore this comment